deployment/human-connection/deployment-neo4j.yaml

```diff
@@ -1,47 +1,61 @@
----
-  apiVersion: extensions/v1beta1
-  kind: Deployment
-  metadata:
-    name: nitro-neo4j
-    namespace: human-connection
-  spec:
-    replicas: 1
-    strategy:
-      rollingUpdate:
-        maxSurge: 0
-        maxUnavailable: "100%"
-    selector:
-      matchLabels:
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    human-connection.org/selector: deployment-human-connection-neo4j
+  name: neo4j
+  namespace: human-connection
+spec:
+  progressDeadlineSeconds: 2147483647
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-11T15:32:39Z" title="Monday, November 11th 2019, 4:32:39 pm +01:00">Nov 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,47 +1,61 @@
----
-  apiVersion: extensions/v1beta1
-  kind: Deployment
-  metadata:
-    name: nitro-neo4j
-    namespace: human-connection
-  spec:
-    replicas: 1
-    strategy:
-      rollingUpdate:
-        maxSurge: 0
-        maxUnavailable: "100%"
-    selector:
-      matchLabels:
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    human-connection.org/selector: deployment-human-connection-neo4j
+  name: neo4j
+  namespace: human-connection
+spec:
+  progressDeadlineSeconds: 2147483647
```


Where does that come from?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-11T18:23:48Z" title="Monday, November 11th 2019, 7:23:48 pm +01:00">Nov 11, 2019</time>_

---


it came from the command `kubectl convert -f deployment-neo4j.yaml --output-version apps/v1`
when I tried to apply the config, it failed since the api has been updated from `extensions/v1beta1`